modules/imgimport/include/csvreading.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CSV_READING_H_INCLUDED
+#define CSV_READING_H_INCLUDED
+
+#include "metadata.h";
+
+struct Metadatalinkedlist{
+    Metadata a;
+    Metadatalinkedlist* next;
+};
+
+/**
+* @brief this is for reading telemetry logs from the plane and store them into a linked list 
+*
+*
+* @param filename the relative path and name of the file
+* @error if the file opening fails, print a fatal log "error: failed to open the cdv file" and return NULL
```

Comment:
Doxygen doesn't support an error identifier. Just put information about errors in the description. 
Detailed descriptions don't have an identifier, just put the detailed description in a paragraph after the brief.